packages/browser/src/index.ts

```diff
@@ -68,6 +70,34 @@ export default class LogtoClient {
     window.location.assign(signInUri);
   }
 
+  public async signOut(postLogoutRedirectUri?: string) {
+    if (!this.idToken || !this.oidcConfig) {
+      return;
+    }
+
+    const { clientId, requester } = this.logtoConfig;
+    const { endSessionEndpoint, revocationEndpoint } = this.oidcConfig;
+    const logtoKey = getLogtoKey(clientId);
+
+    if (this.refreshToken) {
+      void revoke(revocationEndpoint, clientId, this.refreshToken, requester);
+    }
+
+    const url = generateSignOutUri({
+      endSessionEndpoint,
+      postLogoutRedirectUri,
+      idToken: this.idToken,
+    });
+
+    window.location.assign(url);
```

Comment:
nit: `window.location.assign(url);` could leave it as the last statement.

Reply:
Code updated.